src/course.py

```diff
@@ -56,6 +56,9 @@ def get_course_files(self, course_path) -> None:
         files_body = self.course_soup.find_all(class_="card-body")
 
         for item in files_body:
+            """check if card is not a course content, useful for 'Filter weeks' card"""
```

Comment:
```suggestion
            # check if the card is not a course content, useful for `Filter weeks`  card
```

we use docstring in the class/method definition 
for example 
```python
def hello(name : str) -> str:
  """
  saying hello 
   args:
   input name string, name of the person you are greeting,'
  ""'
   return f'hello, {name}'
````
other than that we use normal `#`